### PR TITLE
Python 3

### PR DESCRIPTION
**Pull request by tylrw**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/aioTV/django-rest-swagger/pull/15" target="_blank">ReviewNinja

File: rest_framework_swagger/docgenerator.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def __init__(self, for_user=None, config=None, request=None):
         self.config = config
         self.user = for_user or AnonymousUser()
         self.request = request
-        self._tag_matchers = map(import_string, self.config.get('tag_matchers'))
+        self._tag_matchers = list(map(import_string, self.config.get('tag_matchers')))
         self._operation_filters = map(import_string, self.config.get('operation_filters', []))
```
Comment (rygwdn) [inline rest_framework_swagger/docgenerator.py]: probably `list` here too

File: rest_framework_swagger/introspectors.py [hunk 2]
```diff
@@ -266,7 +260,7 @@ def get_description(self):
         if self.yaml_parser.get_param('replace_docs', False):
             docstring = method_docs
         else:
-            docstring = "\n\n".join(filter(None, [class_docs, method_docs]))
+            docstring = "\n\n".join([_f for _f in [class_docs, method_docs] if _f])
```
Comment (trdyer) [inline rest_framework_swagger/introspectors.py]: im guessing this was auto-generated. but something nicer than `_f` would be desirable.

Comment (tylrw) [inline rest_framework_swagger/introspectors.py]: it's not so bad since the list is visible - will update

**Comment by trdyer** [comment]
+1

**Comment by daynemay** [comment]
:star:

**Comment by daynemay** [comment]
:star:

**Comment by daynemay** [comment]
:star:

**Comment by daynemay** [comment]
+1

**Comment by daynemay** [comment]
:star:

**Comment by daynemay** [comment]
:star:

**Pull request merged**